eventstore/model.go

```diff
@@ -126,6 +126,15 @@ func (m *Model) Find(result interface{}, q *Query) error {
 	})
 }
 
+// FindJSON executes a Query in in JSONMode and returns the result.
```

Comment:
New API for Querys in JSONMode.
Receives a new datastructure to represent the desired query, and return an array of jsons strings with the results.